modules/tensor_mechanics/src/materials/SingleVariableReturnMappingSolution.C

```diff
@@ -44,13 +42,28 @@ validParams<SingleVariableReturnMappingSolution>()
                         "algorithm. Also use same old defaults for relative_tolerance, "
                         "absolute_tolerance, and max_its.");
 
+  // diagnostic output parameters
+  MooseEnum debug_level_enum("none error always", "none");
+  params.addParam<MooseEnum>(
+      "debug_level", debug_level_enum, "When to output Newton solve information");
+  params.addParam<bool>(
+      "output_iteration_info", false, "Set true to output full Newton iteration history");
+  params.addDeprecatedParam<bool>(
```

Comment:
I wonder how long this has already been deprecated? Maybe you can just pull it out now?

Reply:
I'm not that good at pulling out